notebooks/samples/tensorflow/transfer_learning/ai_platform_transfer_learning.ipynb

```diff
@@ -0,0 +1,847 @@
+{
```

Comment:
<p>Ok for now, later we need to update this to show how to model training and prediction in AI Platform</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/122/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Got it; will be updating soon.